sdc/tests/tests_perf/test_perf_utils.py

```diff
@@ -195,13 +195,78 @@ def get_times(f, *args, iter_number=5):
     return exec_times, boxing_times
 
 
+class ResultsDriver:
+    """Base class. Load and dump results."""
+
+    def __init__(self, file_name, raw_file_name=None):
+        self.file_name = file_name
+        self.raw_file_name = raw_file_name if raw_file_name else f'raw_{file_name}'
```

Comment:
It could be replaced with `self.raw_file_name = raw_file_name or f'raw_{file_name}'`